pkg/kubelet/kubelet_pods.go

```diff
@@ -429,7 +429,7 @@ func (kl *Kubelet) GenerateRunContainerOptions(pod *v1.Pod, container *v1.Contai
 		}
 	}
 
-	opts.DNS, opts.DNSSearch, useClusterFirstPolicy, err = kl.GetClusterDNS(pod)
+	opts.DNS, opts.DNSSearch, _, useClusterFirstPolicy, err = kl.GetClusterDNS(pod)
```

Comment:
Seems like resolv.conf  options are not piped into pod yet. Are we planing to do that in follow-up PR?

Reply:
I believe only the `podSandboxConfig.DnsConfig` returned by `generatePodSandboxConfig() `in `pkg/kubelet/kuberuntime/kuberuntime_sandbox.go` is used to generate `resolv.conf`.

This portion of `GetClusterDNS()` does not appear to be affecting the generation of `resolv.conf`, so the options were ignored.  This portion of settings are used to set parameters for launching a container, and not in the container sandbox.

Reply:
Good catch, looked around and confirmed this is correct.